tests/ssl-opt.sh

```diff
@@ -13029,6 +13032,21 @@ run_test    "TLS 1.3: NewSessionTicket: servername negative check, m->m" \
             -s "server state: MBEDTLS_SSL_NEW_SESSION_TICKET" \
             -s "server state: MBEDTLS_SSL_NEW_SESSION_TICKET_FLUSH"
 
+requires_gnutls_tls1_3
+requires_config_enabled MBEDTLS_DEBUG_C
+requires_config_enabled MBEDTLS_SSL_CLI_C
+requires_all_configs_enabled MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE \
+                             MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_EPHEMERAL_ENABLED \
+                             MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_EPHEMERAL_ENABLED
+run_test    "TLS 1.3: NewSessionTicket: early data, m->G" \
```

Comment:
```suggestion
run_test    "TLS 1.3 m->G: EarlyData: basic check, good " \
```

Reply:
Change the title to "TLS 1.3: m->G: EarlyData: with resume session ticket, good"

Reply:
We only support resumption session ticket. `with resumption session ticket` make no sense.  

And nitpick comment .  Better change to "TLS 1.3 m->G:"
`git grep -nw '"TLS 1.3: .*:' ` has 208 testcases. `git grep -nw '"TLS 1.3 .*:'` has 614 test cases. Please ` '"TLS 1.3 .*:'` .

Reply:
I can change the title as what you said. 
But have to say, even we only support resumption session ticket, it's not explicit to the reader from the title, add the "with resumption session ticket" will make it more readable.

Reply:
> We only support resumption session ticket. `with resumption session ticket` make no sense.
> 
> And nitpick comment . Better change to "TLS 1.3 m->G:" `git grep -nw '"TLS 1.3: .*:' ` has 208 testcases. `git grep -nw '"TLS 1.3 .*:'` has 614 test cases. Please ` '"TLS 1.3 .*:'` .

And this title is from "TLS 1.3: NewSessionTicket" test cases, it's strange about the reviewing standard.  Should we meeting to talk about the title standard?

Reply:
That means basic start for early data. If you other words is better, replace it.